apm-agent-plugins/apm-servlet-plugin/src/test/java/co/elastic/apm/agent/servlet/ServletInstrumentationTest.java

```diff
@@ -113,12 +116,18 @@ void testNoopInstrumentation() throws Exception {
 
     @Test
     void testForward() throws Exception {
+        List<String> expectedValues = Arrays.asList("FORWARD", "FORWARD /test");
```

Comment:
The current code would probably yield the name `FORWARD /forward`, but after fixing this should work with a single expected value, not two options.